src/components/LogIn.js

```diff
@@ -47,66 +49,104 @@ class LogIn extends Component {
 
   render() {
     return (
-      <Container style={styles.bg}>
-        <Container style={styles.container}>
-          <Card style={styles.card}>
-            <Image src={ Logo } style={styles.logo} />
-            <br/>
-            <br/>
-            <text style={styles.text}>live. local. music.</text>
-            <br/>
-            <br/>
-            {this.state.user ?
-              <button onClick={this.logout}>Log Out</button> 
-              :
-              <Button center style={styles.google} onClick={this.login}>
-                <Icon center name='google'/>Google</Button>                         
-            }
-
-          </Card>
-        </Container>
-      </Container>
+      <div style={styles.app}>
+        <div style={styles.forward} />
+        <div style={styles.card}>
+          <div style={styles.logoContainer}>
+            <div style={styles.logo} />
+          </div>
+          <div style={styles.catchPhrase}>
+            live. local. music.
+          </div>
+          <div style={styles.buttonContainer}>
+            <div style={styles.googleBtn} onClick={this.login}>
+              {/* <Icon name='google'> */}
+                Google
+              {/* </Icon> */}
```

Comment:
Am I correct that this is commented out code? Commented code and debuggers should be removed before opening PR. You want only the relevant changes.